packages/next/client/index.js

```diff
@@ -265,9 +266,8 @@ export default async ({ webpackHMR: passedWebpackHMR } = {}) => {
     wrapApp,
     err: initialErr,
     isFallback,
-    subscription: ({ Component, props, err }, App) => {
-      render({ App, Component, props, err })
-    },
+    subscription: async ({ Component, props, err }, App) =>
```

Comment:
Seems this `async` / `await` is unnecessary. Can we directly return `render` to skip regenerator?